mentors/julia/m.json

```
@@ -1 +1,7 @@
-[]
+{
+  "github_username": "miguelraz",
+  "name": "Miguel Raz",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "¡Hola! I am a Mexican physics student who loves to use Julia for High Performance Computing. Pull Requests welcome!"}
```

```suggestion
[
  {
    "github_username": "miguelraz",
    "name": "Miguel Raz",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "¡Hola! I am a Mexican physics student who loves to use Julia for High Performance Computing. Pull Requests welcome!"
  }
]
```
Some small errors in the json and some formatting changes.

---

Thanks @rpottsoh